examples/freertos_plus_tcp/z_pub_st.c

```diff
@@ -30,6 +30,7 @@
 
 #define KEYEXPR "demo/example/zenoh-pico-pub"
 #define VALUE "[FreeRTOS-Plus-TCP] Pub from Zenoh-Pico!"
+#define N 2147483647  // max int value by default
```

Comment:
It's safer to use `int n = ~0;` in such a way we always have the guarantee `n` is `max(int)` regardless the actual size of `int`.

Reply:
You are right. Technically it should be an `unsigned int` and not an `int`.